src/lib/Event/ContentOnTheFlyEvents.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUi\Event;
+
+final class ContentOnTheFlyEvents
+{
+    /** @var string */
+    public const CONTENT_CREATE = 'ezplatform.content_on_the_fly.create';
+
+    /** @var string */
+    public const CONTENT_CREATE_PUBLISH = self::CONTENT_CREATE . '.publish';
```

Comment:
Will be hard to find `ezplatform.content_on_the_fly.create.publish` I think that const should not be concatenated here